### DISCUSSION TITLE
Error happened when checking Explore

### DISCUSSION THREAD

**Discussion by sonderlau** [community/community#45118] [Mobile] [answered]
![Screenshot](https://user-images.githubusercontent.com/12829325/213980962-d9d77937-9c9c-45f6-967a-f007716a657d.jpg)

9102:32B6:CD8D3:EC2E7:63CE2E30

Details:

- Device : OnePlus 7 Pro
- OS : ColorOS v12.1
- Android 12
- GitHub App download from Google Play, refreshed app data and reinstalled app
- No problem when I'm using Google Chrome browser to check out GitHub Explore webpage.
- 4G and WIFI both not working

**Comment by iskae** [comment, marked as answer]
Hi @sonderlau 👋 

Thanks so much for reaching out! I have forwarded your request internally to our API engineers and we will be looking into it right away 👍

**Reply by sonderlau** [reply]
I updated GitHub App to the latest version, and it's working now. Thank you for your excellent work.
